### PR TITLE
Todo app loading todos

### PR DESCRIPTION
**Pull request by Igor-Yaremchuk**
[DEMO LINK](https://Igor-Yaremchuk.github.io/react_todo-app-loading-todos/)

### CODE REVIEW

**Comment by Igor-Yaremchuk** [comment]
I have a problem with npm run deploy, that because demolink is not working ((

File: src/components/Todolist.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          )}
+        >
+          <label
+            className="todo__status-label"
+          >
+            <input
+              type="checkbox"
+              className="todo__status"
+              id="todo__selected"
+            />
```
Comment (polosanya) [inline src/components/Todolist.tsx:37]: The input element has a static id attribute. It is not recommended to have the same id for multiple elements. Consider generating a unique id for each todo using todo.id.

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        }}
+      >
+        Completed
+      </a>
+    </nav>
+    {todos.find(todo => todo.completed) && (
+      <button
+        type="button"
+        className="todoapp__clear-completed"
+        onClick={() => {}}
```
Comment (polosanya) [inline src/components/Footer.tsx:64]: What's the purpose of an empty callback?

File: src/components/Todolist.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          </label>
+          {editing
+            ? (
+              <Form
+                title={title}
+                onSubmit={() => {}}
```
Comment (polosanya) [inline src/components/Todolist.tsx:43]: Same empty handler here

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done!

**Pull request closed without merging**